css/fullcalendar.scss

```diff
@@ -213,9 +218,29 @@
 	word-break: break-word;
 	white-space: normal;
 }
-.fc-timeGridWeek-view .fc-daygrid-more-link {
-	word-break: break-all;
-	white-space: normal;
+.fc-timeGridWeek-view {
+	.fc-daygrid-more-link {
+		word-break: break-all;
+		white-space: normal;
+	}
+
+	.fc-day-today {
+		&.fc-col-header-cell {
+			background-color: var(--color-primary-light);
+		}
+
+		.fc-timegrid-col-frame {
+			background-color: var(--color-primary-light);
+		}
+
+		.fc-daygrid-day-frame {
+			background-color: var(--color-primary-light);
+		}
+
+		.fc-event {
+			box-shadow: 0px 0px 0px 1px var(--color-primary-light) !important;
+		}
```

Comment:
I wonder if we can revert this now that we applied the styling globally. Question is if the global styling is wanted because it modifies the today view as well? Probably all views should be looked at because of that?

Reply:
Can you post screenshots of the week / day view please?

Reply:
@dartcafe please delete this code again since @jancborchardt agreed to make it globablly for all views.

```
		.fc-event {
			box-shadow: 0px 0px 0px 1px var(--color-primary-light) !important;
		}
```
and
```
		&.fc-col-header-cell {
			background-color: var(--color-primary-light);
		}
```
is probably still missing from the global styling though...